src/utils.ts

```diff
@@ -17,3 +18,35 @@ export function makeDateFormat(date: Date) {
 export function numberWithCommas(number: number) {
   return number.toLocaleString();
 }
+
+export function getProfileAnimalType(type: string) {
```

Comment:
```suggestion
export function getProfileAnimalType(type: Animals) {
```

`Animals` Enum을 사용하면 더 탄탄한 타이핑이 될 것 같슴다!

Reply:
엥 이렇게 보니 인자로 받는 것과 반환하는 값이 똑같네여...? 그럼 그냥 저 `type`을 그대로 써도 되지 않나유?? 🤔

Reply:
저도 처음에 그렇게 해보려고했는데, 인자로 넘겨주는 더미데이터의 값이 string 타입이라 Animals enum 타입에 적용했을때 에러가 나더라구요! 혹시 다른 방법이 있을까욤,.,??

Reply:
저는 아직 타입스크립트가 넘나 어렵네여;;..

Reply:
```ts
enum Animals {
  Rabbit = '토끼',
  Turtle = '거북이'
}
interface Foo {
  type: Animals;
}

const DUMMY: Foo = {
  type: Animals.Turtle,
};
```

데이터의 인터페이스 자체를 `string`이 아니라 `Animals`로 선언해주시면 됩니다! API에서 던져주는 값도 동일하게 타이핑 해주시면 문제없을거에유!

Reply:
타입스크립트 처음에 시작할 때 많이 헷갈리죠 ㅜㅜㅜㅜ 저도 첨에 제네릭 같은 거 이해 잘 안되서 힘들었었다는.....

Reply:
애증의 ts네여 증말,, 위 내용은 다음 pr에 올려보겠습니다! 감사해요 😄